src/main/java/com/app/controllers/AdminController.java

```diff
@@ -0,0 +1,16 @@
+package com.app.controllers;
```

Comment:
Nên đưa class này vào trong package admin nhé em
và các controller cần phân quyền admin mới truy cập được thì đặt trong package admin là đẹp 
```suggestion
package com.app.controllers.admin;
```